pinot-segment-local/src/test/java/org/apache/pinot/segment/local/utils/TableConfigUtilsTest.java

```diff
@@ -2346,6 +2346,14 @@ public void testValidateInstancePartitionsMap() {
     }
   }
 
+  @Test
+  public void testValidateSkipSegmentPreprocessFlag() {
```

Comment:
This test is for `TableConfigUtils` class. Consider adding a separate test for `TableConfigBuilder`